### DISCUSSION TITLE
Issue Connecting TSML Meetings to the Meeting Guide App

### DISCUSSION THREAD

**Discussion by AAPlacer** [code4recovery/12-step-meeting-list#500] [Q&A] [answered]
**Background:**

aaplacer.org (AA Placer) is the website for CNIA Area 07, District 21 in Northern California. We launched our website in June 2020 and installed the TSML WP plugin to serve as our meeting directory. 

aasacramento.org (CCFAA) is our Intergroup/Central Office; they, too, use the TSML WP plugin as their meeting directory for all the regions they cover within the area, including the area covered by AA Placer. We now have identical meetings posted in our respective TSML meeting directories. CCFAA adopted the TSML WP plugin before AA Placer launched our website. 

**Issue:**

The Meeting Guide app is pulling the bulk of AA Placer meetings from the CCFAA TSML directory. We would like Meeting Guide to connect to AA Placer for all AA Placer meetings. Right now, some connect to AA Placer, while others connect to CCFAA. We have confirmed that for the meetings we share, we have the correct address programmed in each of our directories. 

[Here's an example](https://drive.google.com/file/d/14-dA9X_QPyzG_KMkHT9AzpxSr3j2XQwq/view?usp=sharing) of "what works", or a meeting that connects to AA Placer in the Meeting Guide app. Interestingly, **ALL** Rocklin Fellowship meetings connect to AA Placer in the app. Here's their respective listings on each website:

[Rocklin @ AA Placer](https://aaplacer.org/meetings/rocklin-fellowship-54/)
[Rocklin @ CCFAA](https://aasacramento.org/meetings/rocklin-fellowship-31/)

[Here's an example](https://drive.google.com/file/d/1RM8PL9aYVsyqJEMvUC5P2JEA1B9OvLSm/view?usp=sharing) of what we don't want, an AA Placer meeting connecting to CCFAA. 

[Roseville @ AA Placer](https://aaplacer.org/meetings/roseville-tuesday-night-group-27/)
[Roseville @ CCFAA](https://aasacramento.org/meetings/roseville-tuesday-night-group-13/)

**Question:**

Based on what I've provided thus far, does anyone know of a fix for this? Do you need more information? Is this a known issue? I doubt we're the only District sharing meeting information with their Intergroup. I'd like to understand how other regions are solving this issue. 

All assistance, feedback, etc. appreciated! Thank you for your service! 

Webmaster,
aaplacer.org

**Comment by joshreisner** [comment, marked as answer]
Hi @AAPlacer, this is the support site for TSML - we don't / can't control which meetings are listed in the Meeting Guide app. Please contact GSO at appsupport@aa.org to discuss it.

Their policy is documented here: https://meetingguide.aa.org/duplicates - as it says there, they are able to change listings when there are specific complaints about the data, but you will need to demonstrate that the CCFAA data is incomplete in some way. In the case you point to, it looks like you have a 7th Tradition link that CCFAA does not, so I would mention that in your email to them.

When you're noting specific instances, be sure to include the specific addresses, cities, and location names, because that will help them identify which meetings you want changed.

Please note it is against their policy to blanket-change all listings on the grounds that one group claims it has authority over a specific geography. There are many such areas.

Also note it's possible to import meetings from one data source to another. It might be worth opening a dialogue with CCFAA to see if they would be interested in importing your listings, so that they are better aligned.

**Reply by CCFAAWebmaster** [reply]
CCFAA here and thanks Josh and all of Dev for the terrific  work. Yes we can import so that CCFAA TSML = AAPlacer or we could Add a data source in CCFAA TSML.  I guess we are wondering why how/why does Meeting App see these differences? Where does meeting app pull from (differently) for these examples?

**Reply by joshreisner** [reply]
That's great! I'd refer to the link above to see how the Meeting Guide duplicates filtering works. It goes by address in an attempt not to double-list meetings. But again, that's not the plugin, that's GSO.

If CCFAA can import from the district then the district can reach out to GSO and say "for these cities, CCFAA is now pulling from us, so please update to also get your listings from us" - so long as CCFAA keeps refreshing that feed everything should stay perfectly in sync.
